### PR TITLE
Updated downloads page with CLI instructions

### PR DESCRIPTION
**Pull request by Dalgam**

### CODE REVIEW

**Comment by nathanaelhoun** [comment]
@Dalgam please create another branch than `main` when creating a PR: it is easier for others to locally add your fork as a remote and switch to your branch, and it will cause less troubles updating your fork later if we squash-merge your PR :wink:

File: .gitignore [hunk 1]
```diff
@@ -35,6 +35,7 @@ yarn-error.log*
 *.tsbuildinfo
 
 ### VisualStudioCode ###
+/.vscode
 .vscode/*
 !.vscode/settings.json
 !.vscode/tasks.json
```
[the remark this answers is not in the document]
Comment (Dalgam) [reply .gitignore:41]: Makes sense to me, just copied what we had in ferdium-app but this is better

File: next.config.js [hunk 2]
```diff
@@ -11,7 +11,6 @@ const nextConfig = {
   sassOptions: {
     includePaths: [path.join(__dirname, "styles")],
   },
-  assetPrefix: './',
```
[the remark this answers is not in the document]
Comment (Dalgam) [reply next.config.js:14]: This line broke the reload functionality  when using `yarn dev ` All seems to work on my fork after build and publish, so I think it's fine to remove it. 

The original intention was to simplify import of components, which is done above in the webpack function instead

File: components/CodeBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      document.execCommand("copy");
+      document.removeChild(text);
+      setCopied(true);
+      return;
+    }
+    navigator.clipboard
+      .writeText(props.code)
+      .then(() => {
+        setCopied(true);
+        console.log("Text copied to clipboard...");
```
[the remark this answers is not in the document]
Comment (Dalgam) [reply components/CodeBlock.tsx:27]: It can be removed :)

**Comment by Dalgam** [comment]
> @Dalgam please create another branch than `main` when creating a PR: it is easier for others to locally add your fork as a remote and switch to your branch, and it will cause less troubles updating your fork later if we squash-merge your PR 😉

Will do! Had it that way at first, but things broke down when I had to do some rebase high jinks to make the PR make sense😅

File: components/CodeBlock.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nathanaelhoun) [reply components/CodeBlock.tsx:27]:
```suggestion
```

**Review by nathanaelhoun — APPROVED** [review]
LGTM!

**Comment by nathanaelhoun** [comment]
@allcontributors[bot] please add @cedricroijakkers  and @Dalgam for doc

**Pull request merged**